File: AndroidRCLGroupName/app/src/main/java/com/example/shakil/androidrclgroupname/Model/Person.java
package com.example.shakil.androidrclgroupname.Model;

public class Person {
    private String name, position;
    private int viewType;

    public Person() {
    }

    public Person(String name, String position, int viewType) {
        this.name = name;
        this.position = position;
        this.viewType = viewType;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPosition() {
        return position;
    }

    public void setPosition(String position) {
        this.position = position;
    }

    public int getViewType() {
        return viewType;
    }

    public void setViewType(int viewType) {
        this.viewType = viewType;
    }
}

File: KotlinRCLGroupName/app/src/main/java/com/example/shakil/kotlinrclgroupname/Model/Person.kt
package com.example.shakil.kotlinrclgroupname.Model

class Person {
    var name: String? = null
    var position: String? = null
    var viewType: Int = 0

    constructor()
    constructor(name:String, position: String, viewType: Int){
        this.name = name
        this.position = position
        this.viewType = viewType
    }
}